Kwf/Acl/Component.php

```diff
@@ -89,6 +89,8 @@ public function __construct()
 
         $this->deny('guest', 'kwf_welcome_welcome');
         $this->deny('guest', 'kwf_component_pages');
+        $this->deny('user', 'kwf_welcome_welcome');
+        $this->deny('user', 'kwf_component_pages');
```

Comment:
I have no idea why kwf_component_pages is denied for guest, we don't need that for user